horace_core/Tobyfit/instpars_DGdisk.m

```diff
@@ -24,27 +24,59 @@
 % Get array of instruments
 instruments = header.instruments;
 nrun=instruments.n_runs;
-inst=repmat({instruments{1}},[nrun,1]);
-for i=2:nrun
-    inst{i}=instruments{i};;
-end
 
 % Fill output arguments
 ei=zeros(nrun,1);
 x0=zeros(nrun,1);
 xa=zeros(nrun,1);
 x1=zeros(nrun,1);
-mod_shape_mono=repmat(IX_mod_shape_mono,[nrun,1]);
-horiz_div=repmat(IX_divergence_profile,[nrun,1]);
-vert_div=repmat(IX_divergence_profile,[nrun,1]);
+
 for i=1:nrun
     ei(i)=header.expdata(i).efix;
```

Comment:
Do we not use `experiment_info` from the SQW object these days?

Reply:
now explicitly referenced in the description of the header argument, but that is experiment_info so should be sufficient reference